src/app/shared/models/error-attributes.ts

```diff
@@ -0,0 +1,8 @@
+export interface ErrorAttributes {
```

Comment:
Can we changed this to `ErrorPageAttributes` or similar? I was confused for a minute 😅

Reply:
Yes, for sure! done